### PR TITLE
fix(scrollbar): add scrollbar type's defination

### PR DESCRIPTION
**Pull request by mistery-li**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/scrollbar/src/index.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
 <script lang="ts">
 import { addResizeListener, removeResizeListener } from '@element-plus/utils/resize-event'
 import { addUnit, isArray, isString, toObject } from '@element-plus/utils/util'
-import { computed, defineComponent, nextTick, onBeforeUnmount, onMounted, provide, ref } from 'vue'
+import { computed, CSSProperties, defineComponent, nextTick, onBeforeUnmount, onMounted, PropType, provide, ref } from 'vue'
```
Comment (JeremyWuuuuu) [inline packages/scrollbar/src/index.vue:31]: You need to use this line for type importing, this is the convention of our repo.

```ts
import type { CSSProperties, PropType } from 'vue'
```

Comment (JeremyWuuuuu) [reply packages/scrollbar/src/index.vue:31]: Add this after the last line of importing actual object.

File: packages/scrollbar/src/index.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
 import Bar from './bar.vue'
```
Comment (JeremyWuuuuu) [inline packages/scrollbar/src/index.vue:33]: Here: add it after line#33.

Comment (mistery-li) [reply packages/scrollbar/src/index.vue:33]: OK，i will changed then created a new pr

Comment (mistery-li) [reply packages/scrollbar/src/index.vue:34]: I have a question, I want to delete other commits, what do you suggest?

**Review by jw-foss — APPROVED** [review]

**Pull request merged**